fenix/app/src/main/res/values/strings.xml

```diff
@@ -584,11 +584,17 @@
     <!-- Text for classic wallpapers title. The first parameter is the Firefox name. -->
     <string name="wallpaper_classic_title">Classic %s</string>
     <!-- Text for limited edition wallpapers title. -->
-    <string name="wallpaper_limited_edition_title">Limited Edition</string>
+    <string name="wallpaper_limited_edition_title" moz:RemovedIn="118" tools:ignore="UnusedResources">Limited Edition</string>
+    <!-- Text for artist series wallpapers title. -->
+    <string name="wallpaper_artist_series_title">Artist series</string>
     <!-- Description text for the limited edition wallpapers with learn more link. The first parameter is the learn more string defined in wallpaper_learn_more-->
-    <string name="wallpaper_limited_edition_description_with_learn_more">The new Independent Voices collection. %s</string>
+    <string name="wallpaper_limited_edition_description_with_learn_more" moz:RemovedIn="118" tools:ignore="UnusedResources">The new Independent Voices collection. %s</string>
+    <!-- Description text for the artist series wallpapers with learn more link. The first parameter is the learn more string defined in wallpaper_learn_more-->
+    <string name="wallpaper_artist_series_description_with_learn_more">The Independent Voices collection. %s</string>
     <!-- Description text for the limited edition wallpapers. -->
-    <string name="wallpaper_limited_edition_description">The new Independent Voices collection.</string>
+    <string name="wallpaper_limited_edition_description" moz:RemovedIn="118" tools:ignore="UnusedResources">The new Independent Voices collection.</string>
+    <!-- Description text for the artist series wallpapers. -->
```

Comment:
Please explain what "independent voices collection" refers to here, so this can be translated accordingly